src/main/java/com/petros/bibernate/dao/EntityPersister.java

```diff
@@ -1,40 +1,18 @@
 package com.petros.bibernate.dao;
 
-import com.petros.bibernate.annotation.FetchType;
-import com.petros.bibernate.annotation.OneToMany;
-import com.petros.bibernate.dao.lazy.LazyList;
 import com.petros.bibernate.exception.BibernateException;
+import com.petros.bibernate.util.EntityUtil;
 import lombok.extern.slf4j.Slf4j;
 
 import java.lang.reflect.Field;
 import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.ParameterizedType;
-import java.sql.Clob;
-import java.sql.Connection;
-import java.sql.Date;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.sql.Time;
-import java.sql.Timestamp;
+import java.sql.*;
```

Comment:
I see that only on my project, the wildcard import is prohibited, and this is set in the IDEA settings.